pxtblocks/fields/field_gridpicker.ts

```diff
@@ -188,38 +285,46 @@ export class FieldGridPicker extends Blockly.FieldDropdown implements FieldCusto
             }
 
             if (this.shouldShowTooltips()) {
-                Blockly.browserEvents.conditionalBind(menuItem, 'click', this, this.buttonClickAndClose_);
+                Blockly.browserEvents.conditionalBind(menuItem, 'click', this, () => this.buttonClickAndClose_.bind(this)(value));
```

Comment:
shouldn't need to do both an arrow function and a call to `.bind(this)`. just the arrow function should do it. same below on 351+352

Reply:
Agreed, fixed here [1c0da5d](https://github.com/microsoft/pxt/pull/10571/commits/1c0da5daa210f9a870cb279da5b4fb9d30eb6808)